internal/cassandra.go

```diff
@@ -41,6 +41,16 @@ type Database interface {
 	RemoveFromTagIndex(tagKey, tagValue string, metricKey api.MetricKey) error
 }
 
+type DatabaseGraphiteStore interface {
```

Comment:
The Database isn't exposed in the EvaluationContext. I think we should look at eventually refactoring this, at the very least.